.github/workflows/generate_release_notes.yml

```
@@ -0,0 +1,36 @@
+
+on:
+  push:
+    branches:
+      - staging
+
+jobs:
+  generate_release_notes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)